wafer/templates/404.html

```diff
@@ -0,0 +1,5 @@
+{% extends "wafer/base.html" %}
+{% block content %}
+<h1>Page not found</h1>
+The page you requested cannot be found.
```

Comment:
Can you add i18n, please? (trans/blocktrans)

We try to tag everything that's user-visible for translation.